src/modules/[chain]/gov/[proposal_id].vue

```diff
@@ -56,7 +56,7 @@ store.fetchProposal(props.proposal_id).then((res) => {
   // @ts-ignore
   delete proposalDetail.content.value;
   // when status under the voting, final_tally_result are no data, should request fetchTally
-  if (proposalDetail.status === ProposalStatus.PROPOSAL_STATUS_VOTING_PERIOD) {
+  if (proposalDetail.status === ProposalStatus.PROPOSAL_STATUS_VOTING_PERIOD || proposalDetail.status === "PROPOSAL_STATUS_VOTING_PERIOD") {
```

Comment:
sao có case này đc nhỉ?

Reply:
trước em có fix lỗi không có description của gov detail.
- Cách lấy data của mình `await this.queryClient.gov.proposal(proposal_id);` trong một vài trường hợp không có description.
- Em update thêm cách lấy data của pigpub `wait this.request(this.registry.gov_proposals_proposal_id, {proposal_id,});` 
=> nó dẫn đến case trên